### ISSUE TITLE
Provide an online (out-of-game) API to obtain game data

### ISSUE THREAD

**Issue by samdeane** [samedicorp/dualuniverse-feedback#3]
It would be great to have an official API for game data, such as:

- planet locations
- industry recipes
- market prices (?)

There are community efforts to provide this data (eg dual.sh), but having a single source of truth would be much better.

**Comment by samdeane** [comment]
https://support.dualthegame.com/hc/en-us/requests/34010

**Comment by Wolw3rin3** [comment]
Some Android app with some limited overview of industy, markets etc. would be nice.

**Comment by samdeane** [comment]
Yeah, I am making one for iOS but it is using static data at the moment. It would be great if it had a live data feed.
